homeassistant/components/roomba/sensor.py

```diff
@@ -126,9 +126,7 @@ class RoombaSensorEntityDescription(
         native_unit_of_measurement=AREA_SQUARE_METERS,
         entity_category=EntityCategory.DIAGNOSTIC,
         value_fn=lambda self: (
-            self.run_stats.get("sqft") * 9.29
-            if self.run_stats.get("sqft") is not None
-            else None
+            sqft * 9.29 if (sqft := self.run_stats.get("sqft")) is not None else None
```

Comment:
```suggestion
            None if (sqft := self.run_stats.get("sqft")) is None else sqft * 9.29
```

You could avoid the `not`